packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -151,10 +170,22 @@ export class BlockList extends Component {
 			isReadOnly,
 			shouldShowInsertionPointBefore,
 			shouldShowInsertionPointAfter,
+			containerStyle,
+			getBlockAttributes,
 		} = this.props;
 
+		const attributes = getBlockAttributes( clientId );
+		let columnContainerStyle = {};
+		if ( attributes ) {
+			columnContainerStyle = this.getVerticalAlignmentRemap(
+				attributes.verticalAlignment
+			);
+		}
+
 		return (
-			<ReadableContentView>
+			<ReadableContentView
+				style={ containerStyle && columnContainerStyle }
```

Comment:
Not sure if this is intentional, did you mean `[ containerStyle, columnContainerStyle ]`?

Reply:
Not really, please also refer to [this comment](https://github.com/WordPress/gutenberg/pull/19013#discussion_r375784158)

I pass `containerStyle` only for Columns block and if it's exist I apply additional style with alignment to `ReadableContentView`. The naming here might be better.

Also according to our Slack discussion I will try to move alignment logic outside the block list component

Reply:
this also should be fixed by moving the logic nested in  `getverticalAlingnment` to Column block